src/D2L.Bmx/BmxConfigProvider.cs

```diff
@@ -55,6 +55,18 @@ internal class BmxConfigProvider : IBmxConfigProvider {
 	}
 
 	public void SaveConfiguration( BmxConfig config ) {
+
+		string bmxDirectory = BmxPaths.BMX_DIR;
+		string configFileName = BmxPaths.CONFIG_FILE_NAME;
```

Comment:
on [L70](https://github.com/Brightspace/bmx/pull/305/files#diff-2eb5576e543a9b7cb2ee5a009fd10a20e4bbeb2a3115370f6d75de9403f7deb1R70) `BmxPaths.CONFIG_FILE_NAME` is used directly without the local variable. seems good to me to always do that unless there are other concerns (e.g. perf, race conditions (neither applies here))